### ISSUE TITLE
Remove references to dev-stock-revad

### ISSUE THREAD

**Issue by michielbdejong** [pondersource/dev-stock#20]
It seems `./scripts/init-sciencemesh.sh` is still pulling in pondersource/dev-stock-revad which is no longer used. We can remove that as it just takes time and uses up disk space.

**Comment by michielbdejong** [comment]
In fact that whole script and references to it can probably be removed?

**Comment by michielbdejong** [comment]
Also https://github.com/pondersource/dev-stock/blob/main/scripts/init-beta-sciencemesh.sh#L15 refers to `pondersource/dev-stock-nc1-sciencemesh` instead of `pondersource/dev-stock-nc1-sciencemesh-network-beta` and same for line 18 there.

**Comment by MahdiBaghbani** [comment]
correct, things are a bit out of date as I was only working on enhancement branch lately, I will fix these on main too, so people can use it while enhancement is not merged
